### DISCUSSION TITLE
客户端在异常退出后会关闭所有其它线程

### DISCUSSION THREAD

**Discussion by ws9528** [dotnet/MQTTnet#1321] [Q&A]
客户端在异常退出(网络中断)后会关闭所有其它线程，这导致 网络恢复后自动重新连接MQTT服务器。这个问题要怎么解决？

**Comment by SeppPenner** [comment]
I will translate this to make it readable for all:

**The client closes all other threads after an abnormal exit**

The client shuts down all other threads after an abnormal exit (interruption of the network), which causes the MQTT server to be automatically reconnected when the network is restored. how to solve this problem?
